lexos/static/js/scripts_topword.js

```diff
@@ -1,4 +1,32 @@
 $(function () {
+  // define error message
+  const msg = 'You do not have enough active documents. Please activate at least two documents using the <a href="./manage">Manage</a> tool or <a href="./upload">upload</a> a new document.'
```

Comment:
this line is too long, use line continuation to break it up

Reply:
Fixed.